packages/flutter_tools/test/general.shard/macos/xcode_test.dart

```diff
@@ -133,85 +133,50 @@ void main() {
           expect(fakeProcessManager.hasRemainingExpectations, isFalse);
         });
 
-        testWithoutContext('xcodeVersionSatisfactory is false when version is less than minimum', () {
+        testWithoutContext('version checks fail when version is less than minimum', () {
```

Comment:
I combined the `isRequiredVersionSatisfactory` and `isRecommendedVersionSatisfactory` tests since they had duplicated set ups.